pkg/recipes/terraform/config/config_test.go

```diff
@@ -316,6 +316,7 @@ func Test_AddProviders(t *testing.T) {
 		expectedUCPConfiguredProviders []map[string]any
 		useUCPProviderConfig           bool
 		expectedConfigFile             string
+		checkProviderKeys              bool // checkProviderKeys should be set to true for tests that include multiple 'expectedUCPConfiguredProviders' and have 'useUCPProviderConfig' enabled.
```

Comment:
Based on the description sounds like we can use the value of the other two properties for if condition instead of introducing this new property?

Reply:
haah! yes, it occurred to me a bit later. updated. thanks!